build.sh

```diff
@@ -33,9 +33,13 @@ docker run --rm  -v $VAULT_LOCATION:/root/.vault-token:ro broadinstitute/dsde-to
 gcloud auth activate-service-account --key-file=image-build-account.json
 gcloud auth configure-docker --quiet
 
-docker image build ./$IMAGE_DIR --tag $GCR_IMAGE_REPO/$IMAGE_DIR:$TAG_NAME --tag $GCR_IMAGE_REPO/$IMAGE_DIR:$VERSION \
-    && docker push $GCR_IMAGE_REPO/$IMAGE_DIR:$TAG_NAME \
-    && docker push $GCR_IMAGE_REPO/$IMAGE_DIR:$VERSION
+docker image build ./$IMAGE_DIR --tag $GCR_IMAGE_REPO/$IMAGE_DIR:$TAG_NAME --tag $GCR_IMAGE_REPO/$IMAGE_DIR:$VERSION 
+
+echo "scanning docker image..."
```

Comment:
I wonder if we should add a bit more info to the message, like

```suggestion
echo "Scanning the docker image for any critical security vulnerabilities..."
```